Promise/Promise+Extras.swift

```diff
@@ -9,7 +9,10 @@
 import Foundation
 
 extension Promise {
-    static func all<T>(promises: [Promise<T>]) -> Promise<[T]> {
+
+    /// Wait for all the promises you give it to fulfill, and once they have, fulfill itself
+    /// with the array of all fulfilled values.
+    public static func all<T>(promises: [Promise<T>]) -> Promise<[T]> {
```

Comment:
Yeah, it's standard headerdoc for Swift code, and shows up when you option-click on the symbol in Xcode.